app/grandchallenge/documentation/models.py

```diff
@@ -0,0 +1,136 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import models
+from django.db.models import Max
+from django.utils.translation import gettext_lazy as _
+from django_extensions.db.fields import AutoSlugField
+from simple_history.models import HistoricalRecords
+
+from grandchallenge.core.templatetags.bleach import clean
+from grandchallenge.core.utils.query import index
+from grandchallenge.subdomains.utils import reverse
+
+
+class LevelChoices(models.TextChoices):

```

Comment:
Nitpick but an `IntegerChoice` and a `SmallPositiveIntegerField` would probably make more sense here.